packages/components/src/components/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Meta } from "@storybook/react";
+import React from "react";
+import noop from "lodash/noop";
+import { Button } from "../Button";
+import { Box } from "../../primitives/Box";
+import { Menu, MenuItemProps } from "./Menu";
+
+export default {
+  component: Menu,
+  title: "Components/Menu",
+} as Meta<typeof Menu>;
+
+export const Default = (): JSX.Element => {
```

Comment:
This is an old way of writing stories using CSF2. Can we use the CSF3 methods here please? https://storybook.js.org/docs/react/writing-stories/introduction

Or you could look at packages/components/src/components/Pagination/Pagination.stories.tsx for examples